docs/bylaws.md

```diff
@@ -0,0 +1,399 @@
+
+### I. Quorum
+
+A majority of the Members in good standing shall constitute a quorum at a meeting of Members.
+
+The Members present at a duly called or held meeting at which a quorum is present may continue to transact business until adjournment notwithstanding the withdrawal of enough Members to leave less than a quorum, if any action taken (other than adjournment) is approved by at least a majority of the Members required to constitute a quorum.
+
+In the absence of a quorum, any meeting of Members may be adjourned from time to time by a majority of the Members present, but no other business may be transacted except as provided in the preceding paragraph.
+
+### J. Action Without a Meeting
+
+Any action required or permitted to be taken by the Members may be taken without a meeting, if all Members shall individually or collectively consent in writing to such action. In addition, any action which may be taken at any regular or special meeting of Members may be taken without a meeting if the Cooperative distributes a written ballot to every Member entitled to vote on the matter. That ballot and any related material may be sent by electronic transmission by the Cooperative and responses may be returned to the Cooperative by electronic transmission. That ballot shall set forth the proposed action, provide an opportunity to specify approval or disapproval of any proposal, and provide a reasonable time within which to return the ballot to the Cooperative.
+
+Approval by written ballot pursuant to this section shall be valid only when the number of votes cast by ballot within the time period specified equals or exceeds the quorum required to be present at a meeting authorizing the action, and the number of approvals equals or exceeds the number of votes that would be required to approve at a meeting at which the total number of votes cast was the same as the number of votes cast by ballot.
+
+### K. Record Date
+
+The Board of Directors may fix, in advance, a date as the record date for the purpose of determining the Members entitled to notice of any meeting of Members. Such record date shall not be more than 60 nor less than ten days before the date of the meeting. If no record date is fixed, Members at the close of business on the business day preceding the day on which notice is given or, if notice is waived, at the close of business on the business day preceding the day on which the meeting is held are entitled to notice of a meeting of Members.
+
+## Article IX: Allocations and Distributions
+
+### A. Fiscal Year
+
+The fiscal year of the Cooperative shall end on the last day of December of each year.
+
+### B. Definitions
```

Comment:
Might be nice to make the words bold, if that's allowed.

Reply:
It is possible but I think it might be easier to just leave them as headers so there is less Markdown formatting to deal with when editing the docs.

Reply:
> Thank you for catching all of these typos and formatting issues!

np! If you wouldn't mind marking the comments as resolved if/once you've addressed them that would be helpful!